--- settings.py	2014-02-05 21:16:38.000000000 +0100
+++ settings.py.new	2014-02-05 21:47:47.074859697 +0100
@@ -41,13 +41,12 @@
         'PORT': '',                      # Set to empty string for default.
     },
     'monitor': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "monitor.sqlite"),
-        # The following settings are not used with sqlite3:
-        'USER': '',
-        'PASSWORD': '',
-        'HOST': '',                      # Empty for localhost through domain sockets or '127.0.0.1' for localhost through TCP.
-        'PORT': ''                       # Set to empty string for default.
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'qlproxy_monitor',
+        'USER': 'root',
+        'PASSWORD': 'P@ssw0rd',
+        'HOST': '127.0.0.1',
+        'PORT': '3306'
     },
     'report': {
         'ENGINE': 'django.db.backends.sqlite3',
